app/models/manager_refresh/save_collection/helper.rb

```diff
@@ -34,8 +34,9 @@ def save_inventory_collection!(inventory_collection, association)
         inventory_objects_index[index] = inventory_object
       end
 
-      unique_index_keys = inventory_collection.manager_ref_to_cols
-      unique_db_indexes = Set.new
+      unique_index_keys      = inventory_collection.manager_ref_to_cols
+      unique_db_indexes      = Set.new
+      unique_db_primary_keys = Set.new
```

Comment:
right, for naming conventions, I tried to align unique_db_indexes and unique_db_primary_keys, which are both Set